sdk/kusto/Azure.ResourceManager.Kusto/src/Scripts/CleanupTestResources.ps1

```diff
@@ -0,0 +1,12 @@
+﻿Import-Module Az.Kusto
```

Comment:
I don't think we want this script here. There are two reasons, one is that the test framework will clean up the created resource by default, second is that it's better to keep the folder structure neat and clear. You can save this locally, but there's no need to merge it into the repo.

Reply:
I have to push it to the repository, 
I want those scripts to be shared across all my team members,
I am not going to be the only one on my team in charge of releasing sdk for kusto.

let me know in which directory can I put my scripts (like setup.ps1) 

by the way we had leftovers from previous c# sdk release not weren't cleared 
I had to clear them manually, this is why I have crated this cleanup script

Reply:
moved all scripts to test path and updated readme file as agreed